cli.js

```diff
@@ -41,21 +48,23 @@ function render() {
 
 var st = speedtest({maxTime: 20000});
 
-setInterval(render, 50);
+if (!cli.flags.json) {
+	setInterval(render, 50);
+}
 
 st.once('testserver', function (server) {
 	state = 'download';
-	stats.ping = chalk.cyan(Math.round(server.bestPing) + chalk.dim(' ms'));
+	stats.ping = (cli.flags.json) ? Math.round(server.bestPing) : chalk.cyan(Math.round(server.bestPing) + chalk.dim(' ms'));
```

Comment:
We're now rounding twice. Probably worth putting it in a variable and use it. Same applies for the below events.